### ISSUE TITLE
Incorrect class for creating negative patches

### ISSUE THREAD

**Issue by advcu987** [choosehappy/public#5]
I think that the below function should be called with class = 0, instead it is called with class = 1.

https://github.com/choosehappy/public/blob/af5e0c61d8f1571cb73700ee317b3779cc72bd4f/DL%20tutorial%20Code/5-mitosis/step1_make_patches.m#L52

function definition below:

https://github.com/choosehappy/public/blob/af5e0c61d8f1571cb73700ee317b3779cc72bd4f/DL%20tutorial%20Code/common/func_extraction_worker_w_rots.m#L1

**Comment by choosehappy** [comment]
wow, pulling up some old code here :) 

yup, you appear to be correct, sorry about that. must have imparted the error while cleaning up the code for release. i've made the change to help others along

by the way, newer pipelines using latest technology are available here:
http://www.andrewjanowczyk.com/digital-pathology-classification-using-pytorch-densenet/
http://www.andrewjanowczyk.com/pytorch-unet-for-digital-pathology-segmentation/

**Comment by advcu987** [comment]
Thank you! I think the old stuff is still really valuable! 
I started with your initial implementation, as an exploratory project, will try to build on that and dig deeper with the subsequent technologies.

**Comment by choosehappy** [comment]
the concepts with the old code remain the same, but the challenge will be
that the libraries themselves (e.g., caffe) are no longer maintained

also, since then, A LOT of improvements have been made in the DL space
(e.g., batch normalization, Adam, etc), which caffe doesn't have and thus
the old code didn't contain

the latter code is much better, it converges faster and produces a more
robust result : )
